packages/xod-project/test/patchPathUtils.spec.js

```diff
@@ -8,6 +8,11 @@ import * as Helper from './helpers';
 
 chai.use(dirtyChai);
 
+// Kludge to use instead of "assert.isTrue" / "assert.isFalse"
+// Cause dirtyChai makes it works wrong
+const isTrue = a => assert.equal(a, true);
+const isFalse = a => assert.equal(a, false);
```

Comment:
I filed #1129. To drop the kludge easily once the refactoring will land, maybe it is better to over-monkey-patch `assert.isTrue` and `assert.isFalse` right here and remove the `dirtyChai` from this particular module?

If agree, lets make it and leave a TODO referring the #1129